### ISSUE TITLE
Suggestion: 3rd party type

### ISSUE THREAD

**Issue by anandkumarpatel** [thejsj/auto-kubernetes-lets-encrypt#3]
It might be useful to turn this into a kubernetes 3rd party resource built on top of secrets.

**Comment by thejsj** [comment]
Yeah, that's something I was considering. Maybe it'd be something like "CertClaim" or something. You'd just say what domains you want and what email and the custom resource would take care of creating it.
